app/models/maths_and_physics/dqt_record.rb

```diff
@@ -78,7 +78,7 @@ class DqtRecord
     # :degree_codes      - An array of the claimants degree JAC or HECOS codes.
     # Previously only JAC codes were checked, however changes to the DQT mean subject codes can be either JACS or HECOS codes.
 
-    def initialize(record)
```

Comment:
@nomasprime Please include the documentation lines `2-16` and `20-27` (these I would argue as being essential to include) and `73-79` in the new `app/models/early_career_payments/dqt_record.rb` class.

When we joined the project months ago we had zero ideas about JACS and HECOS codes. These comments are relevant to the file and should not be hidden in a single file. These are what I would consider +ve comments

I possibly would prefer the HECOS and JAC codes to be in their own module / concern and included

Reply:
Deliberating removed because code's pretty self-documenting.

Reply:
@nomasprime please add. If its self documenting why did you not remove from the class I referenced it from? Its self documenting to you because you have been working in this space for months. It is not to others who join the project. 
Think back to when we joined the project, without the comments in the class I referenced I very much doubt it was clear.

Personally I find the documentation helpful. I can respect that we strong disagree on this, but to get my approval I would like included at the very least the references to where JAC/HECOS can be looked up and the policy links. My preference would be only to approve it is is all there.

I firmly believe as developers we should be consistent. Its fine for our views to change over time, but at a single point in time they should be consistent.